pytradfri/VERSION

```diff
@@ -1 +1 @@
-7.0.6
+7.0.7
```

Comment:
Our standard is to bump the version separately from other changes, just before releasing.

Reply:
ok, I will close this for now and investigate further on my local system. I did now have the problems reported in #314.